rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+   <metadata>
+      <description>
+         This ruleset provides analysis that may require individual attention when migrating to newer Hibernate java constructs.
+      </description>
+      <dependencies>
+         <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
+         <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+      </dependencies>
+      <sourceTechnology id="hibernate" versionRange="[,3.9)" />
+      <targetTechnology id="hibernate" versionRange="[4,)" />
+      <targetTechnology id="eap" versionRange="[6,)" />
+      <tag>hibernate</tag>
+      <tag>reviewed-2015-09-23</tag>
+   </metadata>
+
+   <rules>
+      <rule id="hibernate-catchall-00000">
+         <when>
+            <javaclass references="org.hibernate.{package-remainder}.{type}" />
+         </when>
+         <perform>
+            <iteration>
+               <when>
+                  <not>
+                     <has-hint />
+                  </not>
+               </when>
+               <perform>
+                  <classification title="Hibernate" severity="optional" effort="0">
+                    <description>This is a Hibernate type and it will need to be verified for compatibility with the latest Hibernate version.</description>
+                    <tag>catchall</tag>
+                  </classification>
+               </perform>
+            </iteration>
+         </perform>
+         <where param="package-remainder">
+            <matches pattern="(.*)?" />
+         </where>
+         <where param="type">
+            <matches pattern="[^.]+" />
+         </where>
+      </rule>
+   </rules>
+</ruleset>
```

Comment:
we removed Hibernate catchall rule file in https://github.com/windup/windup-rulesets/commit/2d6325c4c9f386227a2b77c7308e4a0d1dcae299 based on WINDUPRULE-135